planner/core/src/main/java/eu/seaclouds/platform/planner/core/HttpHelper.java

```diff
@@ -83,22 +84,48 @@ public String getRequest(String restPath, List<NameValuePair> params){
         }
     }
 
+    public String postInBody(String restPath, String bodyContent){
+        HttpPost post = new HttpPost(prepareRequestURL(restPath));
+        try {
+            StringEntity entity = new StringEntity(bodyContent);
+            post.setEntity(entity);
+            HttpResponse response = httpclient.execute(post);
+            String result = EntityUtils.toString(response.getEntity());
+            return result;
+        }catch (ClientProtocolException e) {
+            return null;
```

Comment:
@szenzaro what do you mean by SeaClouds error handling? HttpHelper is a util class that could define its contract better, I think. Chapter 8 - Effective Java Joshua Block may give you a good idea